schema.json

```diff
@@ -85,38 +93,55 @@
             "$ref": "#/definitions/license"
         },
         "license-url": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL of the license text under which the software or dataset is licensed (only for non-standard licenses not included in the SPDX License List)."
         },
         "preferred-citation": {
-            "$ref": "#/definitions/reference"
+            "$ref": "#/definitions/reference",
+            "description": "When citing the work, cite this reference instead."
+        },
+        "references": {
+            "description": "Reference(s) to other creative works.",
+            "items": {
+                "$ref": "#/definitions/reference"
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
         },
         "repository": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL to the software or dataset in a repository (when the repository is neither a source code repository nor a build artifact repository)."
```

Comment:
Should perhaps define the use cases for this. E.g., I think Zenodo or figshare or other URLs (not DOIs) would go here, as I consider *artifact/binary/release* repository to be things like bintray. Also see

Reply:
Perhaps we can guide people in the right direction with  a few `examples: []`?

Reply:
Yes, how about 
```suggestion
            "description": "The URL to the software or dataset in a repository (when the repository is neither a source code repository nor a build artifact repository).",
            "examples": [
                "https://edoc.hu-berlin.de/handle/18452/23016",
                "https://ascl.net/2105.013"
            ]
```

Reply:
looks good to me